Pavel_Lukin/Lesson2/black-jack.js

```diff
@@ -0,0 +1,94 @@
+"use strict";
+const input = require('input');
+var fs = require('fs');
```

Comment:
Используйте либо var либо let/const
```suggestion
const fs = require('fs');
```